bindings/cpp/src/android.cc

```diff
@@ -0,0 +1,101 @@
+#include <jni.h>
+#include <string>
+#include "defi-wallet-core-cpp/include/android.h"
+#define SECURE_STORAGE_CLASS "com/cronos/play/SecureStorage"
+using namespace std;
+
+JNIEnv *g_env = NULL;
+namespace org {
+namespace defi_wallet_core {
+
+int secureStorageSetJavaEnv(JNIEnv *userenv) {
+  g_env = userenv;
+  return 1;
+}
+
+int secureStorageWriteBasic(JNIEnv *env, string userkey, string uservalue) {
+
```

Comment:
those JNI calls are C or C++ ? just wondering whether it could also throw exceptions etc.

Reply:
it's c++

Reply:
should that function catch the exceptions and do something, or just let it throw to the caller?

Reply:
currently, it throws any exception, caught in caller's side